content/chapters/chapter-1/index.tsx

```diff
@@ -20,6 +20,7 @@ export const metadata = {
   lessons: [
     'genesis-1',
     'genesis-2',
+    'genesis-3',
```

Comment:
I think you will also have to import `genesis-3` to the `content/index.ts` file. Right now I am getting the following error for genesis-3 page:
<img width="1920" alt="Screenshot 2023-03-11 at 2 02 17 AM" src="https://user-images.githubusercontent.com/54861487/224422902-6d3809b6-2b95-48ce-9888-d7d5ccd2c237.png">

Reply:
Thanks @vinayaksh42, I forgot the import there, nice catch.